google-cloud-bigtable/lib/google/cloud/bigtable/table/column_family_map.rb

```diff
@@ -19,9 +19,9 @@ module Google
   module Cloud
     module Bigtable
       class Table
-        # Table::ColumnFamilyMap is a Hash with column_family name and grpc object.
-        # It is used to create instance.
-        # @example Add column family with name and GC rule
+        # Table::ColumnFamilyMap is a hash with a column_family name and gRPC object.
```

Comment:
Is this correct? Does this class really have a "gRPC object"? I don't see one.

We usually consider details like this to be implementation-specific and not part of the public API.

Reply:
From the example it looks like the values in the hash might be `Google::Cloud::Bigtable::GcRule`:

```rb
          #  gc_rule_1 = Google::Cloud::Bigtable::GcRule.max_versions(3)
          #  column_families.add('cf1', gc_rule_1)
```

The docs for a Hash delegate like this should state the acceptable types for the keys and values. For example:

"is a hash accepting string `ColumnFamily` names as keys and `GcRule` objects as values."

Reply:
A GcRule object is different than a gRPC object. We usually consider a "gRPC object" a naked prototype object.